### ISSUE TITLE
Multi-Label Classification with Scikit-MultiLearn

### ISSUE THREAD

**Issue by charles721** [section-engineering-education/engineering-education#3518]
### Proposed title of article
Multi-Label Classification with Scikit-MultiLearn

### Introduction paragraph (2-3 paragraphs):
Multi-label classification methods allow us to classify data sets with more than 1 target variable, a prediction can belong to various labels. There are various methods that should be used depending on the dataset on hand.
Methods for solving Multi-label Classification Problems
1. Problem Transformation
2. Adapted Algorithm
3. Ensemble approaches

1. Problem Transformation
It refers to transforming the multi-label problem into a single-label problem(s) by using
- Binary Relevance: treats each label as a separate single class classification
- Classifier Chains: In this, the first classifier is trained just on the input data and then each next classifier is trained on the input space and all the previous classifiers in the chain.
- Label Powerset: we transform the problem into a multi-class problem with one multi-class classifier is trained on all unique label combinations found in the training data.
2. Adapted Algorithm
adapting the algorithm to directly perform multi-label classification, rather than transforming the problem into different subsets of problems.
3. Ensemble methods.
Incorporate the merits of two previous approaches.

In this tutorial, we shall use all three methods in building our multi-label model using Scikit-Multilearn.
Scikit-multilearn is a BSD-licensed library for multi-label classification that is built on top of the well-known scikit-learn ecosystem.
### Key takeaways: 
1. Difference between Multi-class classification and Multi-label classification.
2. Installing and exploring Scikit-Multilearn.
3. Using all the three approaches in building and training our model.
4. Building a text classification model.
5. Finding the optimal method of the three.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by charles721** [comment]
Hello @lalith1403 and @WanjaMIKE, what is the status of this topic? Kindly advise.

**Comment by lalith1403** [comment]
Great topic 🚀 , make sure it matches the following: 
* Adds value beyond what is in the official docs and what is covered in other blog sites.
* To avoid potential article closure, it should not overlap with existing EngEd articles or incoming topic suggestions.

Please reference any relevant EngEd articles in yours and build a unique project - Approved. @charles721
